### ISSUE TITLE
Import into IMAP account

### ISSUE THREAD

**Issue by sylvainfaivre** [thunderbird/import-export-tools-ng#506]
Using 14.0.1, as far as I understand, there is no way to import mbox files into an IMAP account, you can only import into local folders.

Creating this issue to track this feature.

**Comment by cleidigh** [comment]
@sylvainfaivre 
You are correct. ImportExportTools has never allowed direct imports to imap folders.
It's by design because the stress and possible server communication would make this potentially unreliable.
@cleidigh

**Comment by bentolor** [comment]
Thanks for the clarification @cleidigh ! 

I've the situation where I gathered around 25 years of email in my local Thunderbird archive. Now I want to copy them (including hundres of folders) onto an IMAP server.

Any hints or maybe related tool to achieve this? Or related tool?

I just tried import-export-tools-ng and was expecting an import to IMAP option. I'm also using `imap-backup` which unfortunately only offers export to Thunderbird Archive.
